### ISSUE TITLE
Hardcoded windows path

### ISSUE THREAD

**Issue by genericdata** [GreshamLab/vivaldi#5]
Hardcoded full paths to data. Need to be relative with `inst/extdata`

https://github.com/GreshamLab/vivaldi/blob/e07caa200dc53deae76505f5ac9be394aeb2663c/vignettes/vignette.Rmd#L35-L39
https://github.com/GreshamLab/vivaldi/blob/e07caa200dc53deae76505f5ac9be394aeb2663c/vignettes/vignette.Rmd#L52

**Comment by mohammedkhalfan** [comment]
Data is now included in the package. Need to use this code to get the paths as they will be different on different systems. 

```
# see all available data
system.file("extdata", package = "vivaldi") %>% list.files()
[1] "H1N1.fa"         "reps.csv"        "SegmentSize.csv" "vcfs"

# ex: get full path to reps.csv
system.file("extdata", "reps.csv", package = "vivaldi") 
[1] "C:/Users/mk5636/Documents/R/win-library/4.0/vivaldi/extdata/reps.csv"  
```

**Comment by mohammedkhalfan** [comment]
For `build_rmd()` you probably need to hardcode these to your local path

**Comment by genericdata** [comment]
Fixed with commit e931f5f14019b728f15ab52842c0d16470504429.
